src/WPML_OptionsManager.php

```diff
@@ -537,7 +537,23 @@ public function _LogMenu() {
                 $emailLogList->display();
                 ?>
             </form>
-            <?php $this->displayMP3Banner() ?>
+			<?php
```

Comment:
These changes use tab-based indentation, while the rest of the project uses space-based indentation.
Could you please use space-based indentation as well, matching the coding style of the project?